packages/EasyDoctrine/src/Subscribers/EntityEventSubscriber.php

```diff
@@ -117,7 +118,7 @@ private function filterEntities(array $entities): array
      */
     private function getClearedChangeSet(array $changeSet): array
     {
-        return \array_filter($changeSet, static function (array $changeSetItem) {
+        return \array_filter($changeSet, static function (array|PersistentCollection $changeSetItem) {
```

Comment:
Ok we change the typehint but don't we need to change the logic of the function based on the type given?

Reply:
Nope, as this function is only needed to filter unchanged `DateTimeInterface`

Reply:
ok then I trust you